lib/pages/settings_pages/settings_page.dart

```diff
@@ -298,12 +296,25 @@ class _SettingsPageState extends State<SettingsPage> {
     );
   }
 
+  languagesSortedAlphabetically() {
+    // Alphabet depends on the currently selected language
+    List<Language> sortedLanguages = languageCodes.map((language) {
+      String translatedName = MyLocalizations.of(context, language.isoCode) ?? language.name;;
+
+      return Language(language.isoCode, translatedName);
+    }).toList();
+
+    sortedLanguages.sort((a, b) => a.name.compareTo(b.name));
+
+    return sortedLanguages;
+  }
+
   void _openLanguagePickerDialog() => showDialog(
         context: context,
         builder: (context) => Theme(
             data: Theme.of(context).copyWith(primaryColor: Style.colorPrimary),
             child: LanguagePickerDialog(
-                languages: languageCodes,
+                languages: languagesSortedAlphabetically(),
```

Comment:
If this function does not have to be used anywhere else, I would go for a one-line solution. I was thinking something like:

```dart
languages: languageCodes.map(
	(language) => Language(
		language.isoCode,
		MyLocalizations.of(context, language.isoCode) ?? language.name
	)).toList()..sort(
	        (a, b) => a.name.compareTo(b.name)
        );
```